apps/dashboard/src/modules/admin/views/AdminUserOverView.vue

```diff
@@ -191,19 +191,35 @@ function debounce(func: (skip: number) => Promise<void>, delay: number): (skip:
 
 const apiCall: (skip: number) => Promise<void> = async (skip: number) => {
   await apiService.user
-    .getAllUsers(
-      Number.MAX_SAFE_INTEGER,
-      skip,
-      searchQuery.value.split(' ')[0] || '',
-      isActiveFilter.value,
-      ofAgeFilter.value,
-      undefined,
-      filters.value.type.value || undefined
-    )
-    .then((response) => {
-      totalRecords.value = response.data._pagination.count || 0;
-      allUsers.value = response.data.records;
-    });
+      .getAllUsers(
+          Number.MAX_SAFE_INTEGER,
```

Comment:
I joinked in your other PR to make this a DEFAULT_MAX_PAGINATION

(i know this was my code, but it sucks.)

So maybe check your other PR to see how I fixed it / changed it.

https://github.com/GEWIS/sudosos-frontend/pull/456/files#